Jenkinsfile

```diff
@@ -39,7 +39,6 @@ pipeline {
                         DOCKER_HUB_CREDS = credentials('DockerHubToken')
                     }
                     steps {
-                        sh 'skopeo copy containers-storage:localhost/$IMAGE_NAME docker://docker.io/ucsb/$IMAGE_NAME:latest --dest-username $DOCKER_HUB_CREDS_USR --dest-password $DOCKER_HUB_CREDS_PSW'
```

Comment:
Why was this removed?  It should still be writing to the latest tag on success.  It doesn't on int-test, so I'm thinking this is another artifact of the int-test merger.

Reply:
Yeah I think you're right.